### DISCUSSION TITLE
Should the NAME tag provide additional name delimiters

### DISCUSSION THREAD

**Discussion by Norwegian-Sardines** [fisharebest/gedcom-name#8] [Ideas]
The Current v5.5.1 and v7 specification does not provide for the `NAME` tag to contain additional identifiers and delimiters to identify name parts when only the `NAME` tag is transmitted or processed.

For example:
King Richard the Lion Hearted (How to identify Richard as a given name, with no surname)
Santiago Ramón y Cajal (How to identify multiple independent surnames, **allow /Ramón/ y /Cajal/** ?)

**Comment by hartenthaler** [comment]
I would prefer not to use additional new delimiters to mark the given names. For me, the surname is an important part of the name that allows the grouping and sorting of names. So it makes sense to indicate this name part by using "/". But given names, name prefixes, and all the other name pieces are all less important compared with surnames.

- So I would **not** like to use: NAME King %Richard% the Lion Hearted.
- But I prefer to add the possibility to have more than one surname be highlighted by "/": NAME Santiago /Ramón/ y /Cajal/
- And I'm a fan of indicating the less important and inofficial nick name: NAME Josef "Sepp" /Huber/
- And I would suggest indicating more important name parts by an asterisk attached at the end of a name part: NAME Santiago Salvator* /Ramón*/ y /Cajal/ (so that an application can use "Salvator Ramón" if there is not enough space in order to display the full name or to underline or highlight the more important name parts (Santiago **Salvator** **Ramón** y Cajal).

**Reply by webtrees-pesz** [reply]
Hermann, 
are we sure, that GEDCOM has to highlight 2 surnames in the name term Santiago /Ramón/ y /Cajal/?

I’m not an expert for Spanish names but in my opinion there is only one surname to be highlighted:  Santiago /Ramón y Cajal/
The first part of the surname is (traditionally) the first surname of the person's father, the second part of the surname is the first surname of the person's mother.
Similar in German law, where the family name can consist 2 parts of the name, the surname of the husband and the surname of the wife, or vice versa: Eva /Müller-Schulze/.

Some considerations:

In a future modeling of names in the GEDCOM format, it should be considered to maintain in general the different name parts mandatory in substructures. The NAME tags, on the other hand, should represent the form in which a person's name was found in the cited sources. For output, a new NAME tag should represent the name as it is to be used in lists or diagrams. 

A distinction could also be made here between a long and short form, for use in long or short output fields. This would also make sense for places.

**Comment by Norwegian-Sardines** [comment]
The subtags to `NAME` are not required and in the past making them mandatory has present some major pushback from some providers.

Indexing of surnames for either statistical use or generating extracts, lists, family trees as well as display characteristics like: highlighting, uppercase, compact name, requires distinct surname separation. If the `NAME` tag continues to be the only mandatory element then other considerations for identifying specific name parts need to be discussed and hopefully implemented by all GEDCOM exports.

>I’m not an expert for Spanish names but in my opinion there is only one surname to be highlighted: Santiago /Ramón y Cajal/

While you are correct that Ramón and Cajal come from the father and mother as two different surnames, they are still two surnames. Some times we confuse modern “western” naming standards (data input that only allows one field for a surname) with historical naming customs where honoring ancestors is important.

A woman when married could actually add a third surname to include her husband’s primary surname (inherited from his father in a patriarchal system).

While I can’t speak to German laws, in other countries, a hyphen creates a new surname and is register as such. So the comparison is probably not the same. Laws and usages change over time, a hyphenated name may not have been ‘a thing’ a 100 or 200 years ago!

**Comment by hartenthaler** [comment]
No, "we" are not sure. I'm not an expert on names like "Ramón y Cajal". But I can imagine that it would make sense to sort this person into two surname groups: "Ramón" and "Cajal" and not only in the group "Ramón y Cajal". And maybe one of the two surnames is more important, so highlighting it with an asterisk could make sense.

It could be 

> 1 NAME Santiago /Ramón/ y /Cajal/
> 2 GIVN Santiago
> 2 SURN Ramón
> 2 SURN Cajal  

or 

> 1 NAME Santiago /Ramón y Cajal/
> 2 GIVN Santiago
> 2 SURN Ramón, Cajal, Ramón y Cajal

I don't know what would be better.

**Comment by hartenthaler** [comment]
But it should not be the case that a genealogical program generates from 

> 1 NAME Santiago /Ramón y Cajal/

something internally like 

> 1 NAME Santiago /Ramón y Cajal/
> 2 GIVN Santiago
> 2 SURN Ramón, **y**, Cajal, Ramón y Cajal

because there should be no surname group "y". So to make it clear, I would prefer

> 1 NAME Santiago /Ramón/ y /Cajal/

if name pieces are not mandatory.

**Comment by webtrees-pesz** [comment]
Ok, what does this mean for other surnames like Müller-Schulze? Which emphasis is preferred in this case? Eva /Müller/-/Schulze/?

As long as search forms differentiate name parts, e.g. given name or surname, the indexing of names should be based on clearly defined name tags (like webtrees is doing it) and not of interpretation of a name string.

Therefore, I would favour the mandatory provision of subtags for the name structure in future GEDCOM versions. Everything else is reading in a crystal ball. :-)

**Comment by Norwegian-Sardines** [comment]
> Therefore, I would favour the mandatory provision of subtags for the name structure in future GEDCOM versions. Everything else is reading in a crystal ball. :-)

I’m an advocate of requiring `NAME` subtags to be required for cases where the `NAME` tag itself can not provide enough information for the receiving program to provide services like surname indexing where multiple surnames are customarily used (Spanish, Portuguese), or in area were surnames are made up of a root/stem name and a gender based prefix (Eastern Europe).  I’m sure that other naming customs could benefit from a requirement that a specific `NAME` subtag (beyond surname) be present in a GEDCOM transfer so that vital indexing, grouping or identifying parts are not lost.

Unfortunately, many applications as well as family historians don’t see or understand the need for these indexing services or acknowledge multi-surname or gender based prefixes, and therefore would not want or provide input for these mandatory subtags.
